packages/ui/core-components/src/lib/atoms/inputs/checkbox/Checkbox.svelte

```diff
@@ -27,29 +29,53 @@
 	export let hideDuringPrint = true;
 	$: hideDuringPrint = hideDuringPrint === 'true' || hideDuringPrint === true;
 
+	/** @type {boolean} */
+	export let checked = false;
+
+	/** @type {boolean | string} */
 	export let defaultValue = false;
```

Comment:
Is `checked` replacing `defaultValue`?

If so, we should add a log explaining that `defaultValue` is deprecated.
If not, why do we need `checked`?

Reply:
yes, checked is replacing the default value, will add a log for deprecation